src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const BASE_PRICE = PRICE_PER_DAY * days;
+
+  if (days >= LONG_TERM) {
+    return BASE_PRICE - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return BASE_PRICE - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
```suggestion
  } 
   
   if (days >= SHORT_TERM) {
    return BASE_PRICE - SHORT_TERM_DISCOUNT;
  }
```

Reply:
but why make them separate? If elseif statement makes function run faster

Reply:
how it will make our function work faster if we always check the condition that is in the if
<img width="1017" alt="image" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/70220667/135e77de-0f6b-473e-a9e5-3bec8aee9023">